src/lib/jitter/jitter.h

```diff
@@ -0,0 +1,24 @@
+/*
+* Jitter utility functions interface
+* (C) 2024 Planck Security S.A.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <botan/build.h>
+
+#ifdef BOTAN_HAS_JITTER
+extern "C" {
+   #include <jitterentropy.h>
```

Comment:
This external include should be included in at most one source file, not in the header